tests/upgrades/test_yum_plugins.py:218: TimedOutError

self = <tests.upgrades.test_yum_plugins.Scenario_yum_plugins_count testMethod=test_pre_scenario_yum_plugins_count>

    @pre_upgrade
    def test_pre_scenario_yum_plugins_count(self):
        """Create content host and register with Satellite.
    
        :id: preupgrade-45241ada-c2c4-409e-a6e2-92c2cf0ac16c
    
        :steps:
    
            1. Before Satellite upgrade.
            2. Create LifecycleEnvironment.
            3. Enable/sync 'base os RHEL7' and tools repos.
            4. Create 'Content View' and activation key.
            5. Create a content host, register and install katello-agent on it.
    
        :expectedresults:
    
            1. The content host is created.
            2. katello-agent install and goferd run.
        """
        environment = entities.LifecycleEnvironment(organization=self.org
                                                    ).search(query={'search': 'name=Library'})[0]
        repos = self._get_rh_rhel_tools_repos()
        content_view = self._publish_content_view(org=self.org, repolist=repos)
        ak = entities.ActivationKey(content_view=content_view,
                                    organization=self.org.id,
                                    environment=environment).create()
    
        rhel7_client = dockerize(ak_name=ak.name, distro='rhel7', org_label=self.org.label)
        client_container_id = [value for value in rhel7_client.values()][0]
        wait_for(
            lambda: self.org.name in execute(docker_execute_command,
                                             client_container_id,
                                             'subscription-manager identity',
                                             host=self.docker_vm)[self.docker_vm],
            timeout=300,
            delay=2,
>           logger=self.logger
        )
E       wait_for.TimedOutError: Could not do lambda defined as `lambda: self.org.name in execute(docker_execute_command,
E                                                    client_container_id,
E                                                    'subscription-manager identity',
E                                                    host=self.docker_vm)[self.docker_vm],` at /home/jenkins/workspace/automation-preupgrade-6.6-scenario-tests-rhel7/tests/upgrades/test_yum_plugins.py:212 in time

tests/upgrades/test_yum_plugins.py:218: TimedOutError

--- a/tests/upgrades/test_yum_plugins.py
+++ b/tests/upgrades/test_yum_plugins.py
@@ -213,7 +213,7 @@ class Scenario_yum_plugins_count(APITestCase):
                                              client_container_id,
                                              'subscription-manager identity',
                                              host=self.docker_vm)[self.docker_vm],
-            timeout=300,
+            timeout=500,
             delay=2,
             logger=self.logger
         )

